README.md

```diff
@@ -195,6 +195,7 @@ Settings can be provided from the command line, a configuration file, or an envi
 | default-balances-limit        |         | default-balances-limit        | INDEXER_DEFAULT_BALANCES_LIMIT        |
 | max-applications-limit        |         | max-applications-limit        | INDEXER_MAX_APPLICATIONS_LIMIT        |
 | default-applications-limit    |         | default-applications-limit    | INDEXER_DEFAULT_APPLICATIONS_LIMIT    |
+| data-dir                      | i       | data-dir                      | INDEXER_DATA_DIR                      |
```

Comment:
I left this comment on Shiqi's PR also. Could we add a viper alias so that this is mapped to `INDEXER_DATA`? That would allow us to skip the manual env variable checking.

Reply:
@AlgoStephenAkiki does exporting `INDEXER_DATA_DIR` work for you? after adding the alias, `viper.RegisterAlias("data-dir", "data")`, only `INDEXER_DATA` is working for me.

Reply:
That's a good point.  Do we want both?  Performing the alias I think removes the DATA_DIR variant.

Reply:
I don't think we need both, lets update the docs to refer to `INDEXER_DATA`. Nice catch.